nss/src/runScraper.ts

```diff
@@ -4,37 +4,50 @@ import scrapeBookings from "./scrapeBookings";
 import parseBooking from "./parseBooking";
 import scrapeBuildings from "./scrapeBuildings";
 import { DRYRUN, HASURAGRES_API_KEY, HASURAGRES_URL, YEAR } from "./config";
-import axios from 'axios';
-import { formatString } from './stringUtils';
+import axios from "axios";
+import { formatString } from "./stringUtils";
+import { scrapeRoomFacilities } from "./scrapeRoomAdditionalInformation";
 
 const runScrapeJob = async () => {
   const buildings = await scrapeBuildings();
   const rooms = await scrapeRooms();
+  const facilitiesPromises = rooms.map((room) =>
+    scrapeRoomFacilities(room.id)
+  );
 
   // Filter buildings with no rooms
   const filteredBuildings = buildings.filter(
-    building => !!rooms.find(room => room.id.startsWith(building.id))
+    (building) => !!rooms.find((room) => room.id.startsWith(building.id))
   );
 
-  const bookingPromises = rooms.map(room => scrapeBookings(room.id));
-  const bookings = (await Promise.all(bookingPromises)).flat();
-  const parsedBookings = bookings.map(parseBooking).flat();
+  const bookingPromises = rooms.map((room) => scrapeBookings(room.id));
+  const [facilities, bookings] = await Promise.all([
+    Promise.all(facilitiesPromises),
+    Promise.all(bookingPromises),
+  ]);
```

Comment:
I tried doing the two `Promise.all` separately but I keep getting `socket hang up` error after the first Promise array went through. No idea why it happens lol - if anyone has a clue